### PR TITLE
Link to GitHub help pages for terms in README

### PR DESCRIPTION
**Pull request by Hessel-Otten**
The README stresses thst this repo is intended for forks, but GitHub newbies
may have no idea what these are about, so we link to the official docs!

We provide even more links to GitHub help for newbies: branch management, issues, milestones, labels and wikis.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ A great example repo you can fork to your heart’s content, that serves as a la
 
 ## Getting started
 
-The first thing you want to do with this repo is quite likely **fork it**.  Use the Fork button in the top-right section of the page to do this.
+The first thing you want to do with this repo is quite likely [**fork it**](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).  Use the Fork button in the top-right section of the page to do this.
 
-Once you’re on your fork, you can experiment however you like with it, play with branches, issues, milestones, labels, the wiki, and more.
+Once you’re on your fork, you can experiment however you like with it, play with [branches](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-branches#about-managing-branches), [issues](https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#issues), [milestones](https://docs.github.com/en/github/managing-your-work-on-github/about-milestones), [labels](https://docs.github.com/en/github/managing-your-work-on-github/managing-labels#about-labels), the [wiki](https://docs.github.com/en/github/building-a-strong-community/about-wikis), and more.
```
Comment (Werknemer) [inline README.md:10]: I believe https://help.github.com/articles/viewing-branches-in-your-repository/ would be a better entry point for branche help 😉

Comment (Hessel-Otten) [reply README.md:10]: He Hes!

Thanks for helping improve the README. Overall it looks great, there's just a litle link change it like you to consider

**Comment by Hessel-Otten** [comment]
Verzonden vanuit Mail<https://go.microsoft.com/fwlink/?LinkId=550986> voor Windows 10

Van: Hes<mailto:notifications@github.com>
Verzonden: zaterdag 6 februari 2021 09:20
Aan: Hessel-Otten/oreilly-github-demo<mailto:oreilly-github-demo@noreply.github.com>
CC: Hessel<mailto:h.j.otten@outlook.com>; Assign<mailto:assign@noreply.github.com>
Onderwerp: Re: [Hessel-Otten/oreilly-github-demo] Link to GitHub help pages for terms in README (#1)


@Werknemer commented on this pull request.

Hey, that’s a great point! I’ll do that.

**Comment by Werknemer** [comment]
Oh thanks! Glad you like it!

Verzonden vanuit Mail voor Windows 10

Van: Hessel
Verzonden: zaterdag 6 februari 2021 09:26
Aan: Hessel-Otten/oreilly-github-demo
CC: Hes; Assign
Onderwerp: Re: [Hessel-Otten/oreilly-github-demo] Link to GitHub help pages forterms in README (#1)

@Hessel-Otten commented on this pull request.

In README.md:
>  

-Once you’re on your fork, you can experiment however you like with it, play with branches, issues, milestones, labels, the wiki, and more.

+Once you’re on your fork, you can experiment however you like with it, play with [branches](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-branches#about-managing-branches), [issues](https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#issues), [milestones](https://docs.github.com/en/github/managing-your-work-on-github/about-milestones), [labels](https://docs.github.com/en/github/managing-your-work-on-github/managing-labels#about-labels), the [wiki](https://docs.github.com/en/github/building-a-strong-community/about-wikis), and more.

He Hes!
Thanks for helping improve the README. Overall it looks great, there's just a litle link change it like you to consider

**Comment by Werknemer** [comment]
👍🏻

**Pull request merged**